ext/sockets/sockaddr_conv.c

```diff
@@ -13,7 +13,7 @@ extern zend_result php_string_to_if_index(const char *val, unsigned *out);
 
 #ifdef HAVE_IPV6
 /* Sets addr by hostname, or by ip in string form (AF_INET6) */
-int php_set_inet6_addr(struct sockaddr_in6 *sin6, char *string, php_socket *php_sock) /* {{{ */
+zend_result php_set_inet6_addr(struct sockaddr_in6 *sin6, char *string, php_socket *php_sock) /* {{{ */
```

Comment:
You could use `bool` here instead of `zend_result` this would preserve the existing semantics

Reply:
I did that at first but was to make it consistent with the callers [e.g. here](https://github.com/php/php-src/pull/17297/files#diff-d650448fb6c4b8f7db479b78b11bbaff1f275fe7b2ad15f52abfb625ac12a442L133).

Reply:
How public are these APIs? Because it might make more sense to change the callers API to return a `bool` (which is `static` here).

Reply:
fair points